lib/src/material_tooltip/tooltip_source.dart

```diff
@@ -24,16 +24,22 @@ const tooltipShowDelay = const Duration(milliseconds: 600);
 ///
 /// NOTE: This directive will be used in conjunction with tooltips once they are
 /// made to be popups instead of just using popups.
-@Directive(selector: '[tooltipSource]', exportAs: 'tooltipSource', host: const {
-  '(mouseover)': 'onMouseOver()',
-  '(mouseleave)': 'onMouseLeave()',
-  '(focus)': r'onFocus()',
-  '(blur)': 'onBlur()',
-  '(keyup)': 'onKeyUp()',
-  'tabindex': '0',
-  'aria-label': 'tooltipLabel',
-  '[attr.style]': "'cursor: pointer'"
-})
+@Directive(
+  selector: '[tooltipSource]',
+  exportAs: 'tooltipSource',
+  host: const {
+    '(mouseover)': 'onMouseOver()',
+    '(mouseleave)': 'onMouseLeave()',
+    '(focus)': r'onFocus()',
```

Comment:
Probably don't need the raw string qualifier here.

Reply:
We are starting to clean up the `host:` properties of the `@Component` and `@Directive` annotations in the library so these will probably go away in the near future.